draft/spec/index.html

```diff
@@ -868,6 +868,28 @@ <h2>Extensions</h2>
             </p>
         </blockquote>
     </section>
+    
+    <section id="filesystem-features">
+        <h2>Filesystem features</h2>
+        <p>
+            In order to maximize the compatibility of the OCFL with different filesystems, and thus improve the
+            portability of OCFL Objects between different systems, some restrictions on the use of certain
+            filesystem features are necessary. If the preservation of non-OCFL-compliant features is required then
+            data MUST be wrapped in a suitable disk or filesystem image format which OCFL can treat as a regular
+            file.
+        </p>
+        <ol>
+          <li>Filesystem metadata (e.g. permissions, access, and creation times) are not considered portable between
+              filesystems or preservable through file transfer operations. Nor can these attributes be validated in
+              terms of fixity in a consistent manner. As such, the OCFL does not support the portability of these
+              attributes.</li>
+          <li>Hard and soft (symbolic) links are not portable and MUST NOT be used within OCFL Storage hierachies.
+              A common use case for links is storage deduplication. OCFL inventories provide a portable method
```

Comment:
This seems to start making a point and then drifts off.

Reply:
The point seems succinct to me. Are you suggesting we add something along the lines of "and so links are not necessary"?

Reply:
IMO it would be useful to know how inventories achieve the same effect; that's not spelled out in the spec, so people are left to fill in the gaps.

Reply:
Would language like:

> OCFL inventories provide a portable method of achieving the same effect by using hashes to address content.

Reply:
s/hashes/digests/ but yes